docs/ide-usage.md

```diff
@@ -64,9 +64,9 @@ The following plugins are needed:
      Configure for all the languages used in TEAMMATES:
      * Java: `Java → Code Style → Formatter → Edit → Tab policy → Spaces only`.
      * JavaScript: `JavaScript → Code Style → Formatter → Edit → Tab policy → Spaces only`.
-     * HTML: `Web → HTML Files → Editor → Indent using spaces`.
-     * CSS: `Web → CSS Files → Editor → Indent using spaces`.
-     * XML: `XML → XML Files → Editor → Indent using spaces`.
+     * HTML: `Eclipse → Preferences → Web → HTML Files → Editor → Indent using spaces`.
+     * CSS: `Eclipse → Preferences → Web → CSS Files → Editor → Indent using spaces`.
+     * XML: `Eclipse → Preferences → XML → XML Files → Editor → Indent using spaces`.
```

Comment:
I think it would be better to add an additional statement about it right before this part, since the reader would probably skim through that line or forgot about where they were supposed to go by the time they reached this section.